handlers/main.yml

```diff
@@ -1,10 +1,10 @@
 ---
 - name: Restart
-  command: /sbin/shutdown -r +0 'Ansible Reboot triggered'
-  async: 0
+  shell: sleep 2 && /sbin/shutdown -r now 'Ansible Reboot triggered'
```

Comment:
What's the reason for waiting two seconds before restarting?

Reply:
Refer to this comment https://github.com/ansible/ansible/issues/14413#issuecomment-182470142

Reply:
OK, strange but makes sense...